crates/services/executor/src/hacks.rs

```diff
@@ -0,0 +1,133 @@
+use fuel_core_storage::{
+    column::Column,
+    kv_store::KeyValueInspect,
+    tables::ContractsState,
+    transactional::StorageTransaction,
+    ContractsStateKey,
+    StorageAsMut,
+};
+use fuel_core_types::{
+    fuel_tx::Input,
+    fuel_types::{
+        Bytes32,
+        ContractId,
+    },
+    services::executor::Result as ExecutorResult,
+};
+
+struct Slot {
+    key: &'static str,
+    value: &'static str,
+}
+
+const CONTRACT_ID_WITH_BAD_STATE: &str =
+    "0x7e2becd64cd598da59b4d1064b711661898656c6b1f4918a787156b8965dc83c";
+
+const STATE: [Slot; 4] = [
+    Slot {
+        key: "35fa5b7532d53cf687e13e3db014eaf208c5b8c534ab693dd7090d5e02675f3e",
+        value: "0000000000000000000000000000000000000000000000000000000000000000",
+    },
+    Slot {
+        key: "35fa5b7532d53cf687e13e3db014eaf208c5b8c534ab693dd7090d5e02675f3f",
+        value: "0000000000000000000000000000000000000000000000000000000000000000",
+    },
+    Slot {
+        key: "7bb458adc1d118713319a5baa00a2d049dd64d2916477d2688d76970c898cd55",
+        value: "0000000000000000000000000000000000000000000000000000000000000000",
+    },
+    Slot {
+        key: "7bb458adc1d118713319a5baa00a2d049dd64d2916477d2688d76970c898cd56",
+        value: "0000000000000000000000000000000000000000000000000000000000000000",
+    },
+];
+
+pub fn maybe_fix_storage<S>(
+    inputs: &[Input],
+    storage: &mut StorageTransaction<S>,
+) -> ExecutorResult<()>
+where
+    S: KeyValueInspect<Column = Column>,
+{
+    for input in inputs {
+        if let Input::Contract(contract) = input {
+            maybe_fix_contract(&contract.contract_id, storage)?;
+        }
+    }
+    Ok(())
+}
+
+pub(crate) fn maybe_fix_contract<S>(
+    contract_id: &ContractId,
+    storage: &mut StorageTransaction<S>,
+) -> ExecutorResult<()>
+where
+    S: KeyValueInspect<Column = Column>,
+{
+    let bad_contract_id: ContractId = CONTRACT_ID_WITH_BAD_STATE.parse().unwrap();
+
+    if contract_id == &bad_contract_id {
+        for slot in STATE.iter() {
+            let storage_key: Bytes32 = slot.key.parse().unwrap();
+
+            let key = ContractsStateKey::new(contract_id, &storage_key);
+            let contains = storage
+                .storage_as_mut::<ContractsState>()
+                .contains_key(&key)?;
```

Comment:
qq: what was the previous value stored in the keys defined by `STATE`? does zeroing them out fix it?